docs/upgrade.txt

```diff
@@ -201,73 +198,63 @@ demonstrates how to execute a mapReduce command using the
    ]);
    
    $resultDocument = $cursor->toArray()[0];
-   
-   ?>
 
 DBRef Helpers
-~~~~~~~~~~~~~
+-------------
 
-:phpclass:`MongoDB\\Collection` does not yet
-have helper methods for working with
-:manual:`DBRef </reference/database-references>`
-objects; however, that is planned in
-:issue:`PHPLIB-24`.
+:phpclass:`MongoDB\\Collection` does not yet have helper methods for working
+with :manual:`DBRef </reference/database-references>` objects; however, that is
+planned in :issue:`PHPLIB-24`.
 
 MongoCollection::save() Removed
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+-------------------------------
 
-:php:`MongoCollection::save() <mongocollection.save>`,
-which was syntactic sugar for an insert or upsert operation, has been
-removed in favor of explicitly using
+:php:`MongoCollection::save() <mongocollection.save>`, which was syntactic sugar
+for an insert or upsert operation, has been removed in favor of explicitly using
 :phpmethod:`MongoDB\\Collection::insertOne` or
-:phpmethod:`MongoDB\\Collection::replaceOne` with the ``upsert``
-option).
+:phpmethod:`MongoDB\\Collection::replaceOne` (with the ``upsert`` option).
 
-.. .. figure:: img/save-flowchart.png
+.. .. figure:: /images/save-flowchart.png
```

Comment:
@schmalliso: When I uncommented this, I received a build warning for `image file not readable: images/save-flowchart.png`. Any ideas?

On a side note, I also get another build warning for `logo file '.static/logo-mongodb.png' does not exist`. Is this related to the empty `/docs/.static/.mongodb` file? Should we leave that in place for the upstream build process?